be/src/olap/rowset/segment_v2/binary_plain_page.h

```diff
@@ -158,6 +166,7 @@ class BinaryPlainPageBuilder : public PageBuilder {
     uint32_t _last_value_size = 0;
     faststring _first_value;
     faststring _last_value;
+    bool _is_dict_page;
```

Comment:
move it to `PageBuilderOptions` is better?

Reply:
indeed!